sql/server.go

```diff
@@ -151,8 +153,17 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 type parameters []driver.Datum
 
 // Arg implements the Args interface
-func (p parameters) Arg(i int) (parser.Datum, bool) {
-	if i < 1 || i > len(p) {
+func (p parameters) Arg(name string) (parser.Datum, bool) {
+	if !unicode.IsDigit(rune(name[0])) {
```

Comment:
We can't use `unicode.IsDigit` here unless we also parse the number with a unicode-aware function. `strconv.ParseInt` only supports ASCII digits. `rune(name[0])` is also incorrect for non-ASCII characters; we must use functions in the `utf8` package instead.  I don't think we want to go down this path; even languages like Go that allow unicode identifiers still require that numbers use ASCII digits.